### PR TITLE
feat: v36 styles

### PR DESCRIPTION
**Pull request by limpbrains**
# Description

- new 32px padding for all Onboarding screens
- Caption13Up is now used in section headers on main screen

---

## Type of change

Bug fix

## Tests
No test

## QA Notes

Nothing to test here, these are just small style changes

### CODE REVIEW

File: src/screens/BetaRisk.tsx
Comment (pwltr) [inline src/screens/BetaRisk.tsx:1]: Color of background glow should be "yellow"

**Review by pwltr — CHANGES_REQUESTED** [review]

**Comment by pwltr** [comment]
Spacing on home screen looks off compared to the design. Cards are too big and the captions are slightly off for total balance and widgets. Probably nothing introduced by this PR but maybe a good place to fix it?

Screenshot 2023-04-12 at 12 35 52

**Comment by limpbrains** [comment]
image

I've tunned spacings a little more, looks good now. Cards size in another PR

File: src/screens/BetaRisk.tsx
Comment (limpbrains) [reply src/screens/BetaRisk.tsx:1]: fixed

**Review by pwltr — APPROVED** [review]

**Pull request merged**